### DISCUSSION TITLE
Display waypoint names in navigation

### DISCUSSION THREAD

**Discussion by Cer-o** [devemux86/cruiser#224] [Q&A] [answered]
Hello, I am en route and sadly I see that the names of the waypoints have not been imported or are not displayed.
I have designed the route with kurviger and I have bothered to name many points of interest, but now that I am on the route I sadly see that the names are not shown.
Do I have to configure something in Cruiser or is it that they can't be displayed?
Thank you

**Comment by WolfgangZ42** [comment]
Just tested it:

- if from kurviger.de exported in kurviger format - waypoint names are there
- if exported as gpx file you hopefully have selected "route" and not only "track", then import it with "route" and it works (automatic or track are not sufficient!)

<img width="200" src="https://github.com/devemux86/cruiser/assets/121806238/622f5387-df00-46c5-85a6-4882eb6384f0"/> - - - -<img width="200" src="https://github.com/devemux86/cruiser/assets/121806238/e8cd50ef-064a-4089-b6c8-3d0fc5a8b515"/>

**Comment by Cer-o** [comment]
Hello, I export it in .kurviger format, but the names do not appear.
Maybe I have to modify something in Cruiser

**Reply by WolfgangZ42** [reply]
Strange - try this .kurviger file and see what happens: [Puchheim - Odenwald.zip](https://github.com/devemux86/cruiser/files/11855800/Puchheim.-.Odenwald.zip)

**Reply by WolfgangZ42** [reply]
Maybe you can send me your file, just to check its content.

**Comment by devemux86** [comment]
@Cer-o everything in the Kurviger routes is imported.

Waypoints names appear in the waypoints list, in waypoint info bubbles, announced in navigation, etc.

- Do you have a sample .kurviger file?

- Can you describe in more detail what steps you take so that we can help with your import process?

Please see below how to import Kurviger routes and how @WolfgangZ42's route appears in navigation:

<img src="https://github.com/devemux86/cruiser/assets/3484020/7337b894-3c2f-40c3-a715-92a39226d66d" width="50%"/><img src="https://github.com/devemux86/cruiser/assets/3484020/0d6fe045-a48a-41b4-8915-d06c9c1d5509" width="50%"/>

**Comment by Cer-o** [comment, marked as answer]
I think I have found the solution, I have activated this box and the name of the waypoint already appears.
![Screenshot_20230624-173230439~2](https://github.com/devemux86/cruiser/assets/122309655/49d384d6-be3e-4aef-ad0e-b6c2a96a1fc9)

**Reply by devemux86** [reply]
@Cer-o did you mean to see the name of the next waypoint in the navigation?
Indeed this is the correct setting.

You can select which waypoint to display with _long press on the waypoint panel_. :slightly_smiling_face:

**Comment by Cer-o** [comment]
Indeed I have named the corresponding waypoints with gas stations and some with cities.

Since you have named the waypoints panel, where can I find the list of waypoints?  in the event that you want to delete one in particular, without having to stop browsing.  .

**Reply by devemux86** [reply]
> where can I find the list of waypoints? in the event that you want to delete one in particular, without having to stop browsing.

The waypoints list is in the menu "Routes | Waypoints".

See all available gestures here:
(drag to reorder, swipe to delete, tap to change, long press for menu)
- https://github.com/devemux86/cruiser/discussions/114#discussioncomment-5559274

Or you can long press a waypoint on the map and use its actions.

**Comment by Cer-o** [comment]
Perfect, I had it in front of me and I hadn't seen it.  I am very satisfied with the service that Cruiser is providing me.  
Thank you very much for your help.
